projects/plugins/jetpack/_inc/lib/admin-pages/class.jetpack-admin-page.php

```diff
@@ -354,6 +354,12 @@ public static function wrap_ui( $callback, $args = array() ) {
 			?>
 			<!-- END OF CALLBACK -->
 
+			<div id="jp-stats-report-bottom">
+				<div class="wrap">
```

Comment:
We have an existing `jetpack_show_promotions` filter that I think we could use here, so folks that already use that filter benefit from it and do not see the new CTAs.

Reply:
Is this what you meant? https://github.com/Automattic/jetpack/pull/20310/commits/7a97c4a4f8c4eec42b3cb9395e1626893b19e77f

Reply:
Yep, thank you!